study-builder/studies/osteo/study.conf

```diff
@@ -166,8 +166,24 @@
       ]
     }
   ],
-
   "activities": [
```

Comment:
I understand why we're editing the `study.conf` file directly. This works fine when we're running the study on our local machines, but this won't work for deployed environments. Currently, study-builder CLI does not have functionality to figure out the edits we made to this `study.conf` and applying to this to production environment where the study already exists. Just want to point this out so we start thinking about how to get this new changes into an existing running study. The way we had been doing it is using "patches" and custom Java code.

Reply:
I started creating a patch and then reading the code I realized I could use the existing `--only-activity` and `--only-events` command line flag. It worked locally and on dev. You still think patch needed?

Reply:
Using the `--only-activity` flag should be fine. But looking at code for `--only-events` it doesn't look like it's smart enough to know what's new and what's not, so I think we'll end up with duplicated events?

Reply:
Looks like this will be addressed in https://github.com/broadinstitute/ddp-study-server/pull/673. Resolving this comment for now.